### ISSUE TITLE
Remove json3 dependency

### ISSUE THREAD

**Issue by gajus** [keen/keen-tracking.js#38]
It is not up to `keen-tracking` to attempt to patch the working environment. `json3` can be a [peer dependency](https://docs.npmjs.com/files/package.json#peerdependencies), but it really shouldn't be even that.

Add a note to the `./README.md` that for `keen-tracking` to work, `window.JSON` must be present.

Now you are adding an extra 42.28kB to everyones bundle because some (what? I don't know a single browser that does not implement `window.JSON`) browsers do not implement it.

**Comment by gajus** [comment]
For the record, I have worked around using `webpack` [`resolve.alias`](https://webpack.github.io/docs/configuration.html#resolve-alias) configuration:

```js
resolve: {
    alias: {
        json3: path.resolve(__dirname, './src/polyfills/json3')
    }
},
```

where `./src/polyfills/json3` is:

```js
export default {
    parse () {
        throw new Error('Client does not implement window.JSON.');
    },
    stringify () {
        throw new Error('Client does not implement window.JSON.');
    }
};
```

**Comment by dustinlarimer** [comment]
@gajus we've always included JSON (and now JSON3) in our SDK to ensure IE6/7 support for recording events, as a surprising number of our customers' users were still active on these versions for awhile. Global usage of those versions has dropped to a point where it seems ok moving forward without this, but at one time we did have to patch the working environment to ensure that it "just worked." 

Thanks for the suggestion here, I do love ripping code out and moving things forward :+1:

**Comment by dustinlarimer** [comment]
Resolved in v1.0 - thanks again!
